Dockerfile

```diff
@@ -59,30 +63,26 @@ RUN apt-get update && \
     libsasl2-dev \
     unzip \
     libsasl2-modules-gssapi-mit && \
-    apt-get clean && \
-     rm -rf /var/lib/apt/lists/*
```

Comment:
I know it's wrong, but can you put the extra space before rm back so that it matches in the diff?